### ISSUE TITLE
Leather Wallet lost the display domain or subdomain name when they are in punycode

### ISSUE THREAD

**Issue by paradigma-cl** [blockstack/blockstack-app#4815]
When you want to select an account, a list of your accounts is displayed.  If an account does not have a domain or subdomain associated to it displays Account 6, otherwise displays the domain or subdomain name.  
Since some weeks ago, the accounts that use punycodes are not displayed at all. Not even the unicode code. 
The same happens to the Dropdown list box of the UI.
It should display the account name.

**Comment by 314159265359879** [comment]
Thanks for reporting, I am seeing reports that recent domain registrations are not displayed at all too.

**Comment by 314159265359879** [comment]
@paradigma-cl which version of the wallet are you using?
 
 Would you mind sharing some screenshots? feel free to blurr private details. 
 
 Do you see any signs in the console that there are issues, reaching relevant API's?
 
![image](https://github.com/leather-wallet/extension/assets/33360391/b25bc7e2-4796-4e8e-af23-e8fba0349e92)
![image](https://github.com/leather-wallet/extension/assets/33360391/0f7004c3-95b0-4940-9cd0-8315bddf284c)

**Comment by 314159265359879** [comment]
were the affected domains registered or transferred recently?

**Comment by paradigma-cl** [comment]
> @paradigma-cl which version of the wallet are you using?
> 
> Would you mind sharing some screenshots? feel free to blurr private details.
> 
> Do you see any signs in the console that there are issues, reaching relevant API's?
> 
> ![image](https://private-user-images.githubusercontent.com/33360391/297707446-b25bc7e2-4796-4e8e-af23-e8fba0349e92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2ODk3MzksIm5iZiI6MTcwNTY4OTQzOSwicGF0aCI6Ii8zMzM2MDM5MS8yOTc3MDc0NDYtYjI1YmM3ZTItNDc5Ni00ZThlLWFmMjMtZThmYmEwMzQ5ZTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE5VDE4MzcxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlNzk2YjlhMWIxMjcyYjc2MGM5NTIwYTBiNGY2ZjNmNjIxMDY4ZWI1YzJmOTFkOGM0YzBkYWYxMWNkNDY5ZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9SCqbZHkAXy25Aw0tQISSIvJorjIxJBLjLTBMb0F6hc) ![image](https://private-user-images.githubusercontent.com/33360391/297707915-0f7004c3-95b0-4940-9cd0-8315bddf284c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2ODk3MzksIm5iZiI6MTcwNTY4OTQzOSwicGF0aCI6Ii8zMzM2MDM5MS8yOTc3MDc5MTUtMGY3MDA0YzMtOTViMC00OTQwLTljZDAtODMxNWJkZGYyODRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE5VDE4MzcxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA0ODQ2ZWM0YTEyYjk3ZTUzYTNkNWM1YmY4Zjk1ZWM2OGEzZDc0ZWQ3N2YxZWYzYzk3MjJjYmZiZmMyZGZmNDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.23BUwbjX7AN0K5TbJpksH4nhIpNCjeyDXj9rOXM4A7s)

![leather_capture_punycode](https://github.com/leather-wallet/extension/assets/32182666/42a37dde-079f-4a0c-9a62-8ceeae3cb6ab)
Above is the Firefox wallet.  It does not display the account name, as it is punycode.
Below is the Chrome wallet
![imagen](https://github.com/leather-wallet/extension/assets/32182666/9c27e975-1cf5-44b6-bea2-90da22eec1aa)

**Comment by paradigma-cl** [comment]
No changes to the names have been made.

**Comment by paradigma-cl** [comment]
I found that something happened with the name, and the stacks address lost its association to the name.  I traced it and could not find a reference anywhere.  Something similar happened jan 2023.  Several accounts lost their associations.  Mainly I think they are subdomain names.
In my case, I saw that the name dissappeared while looking at the Wallet.  There is no problem with the display of the punycode name
